c_libs/vesc_c_if.h

```diff
@@ -556,6 +556,9 @@ typedef struct {
 
 	// Store backup data
 	bool (*store_backup_data)(void);
+
+	// PPM
+	float (*app_ppm_get_servo_val)(void);
```

Comment:
This has to be added to the firmware as well, otherwise it won't work.

Reply:
[Here's the PR for this feature (PPM Input Fetching) as well as the same for UART.](https://github.com/vedderb/bldc/pull/575)

Be sure to check over the logic included for app_ppm.c when config.ctrl_type == PPM_CTRL_TYPE_NONE, as it forces the next iteration of the main loop once the throttle input is gathered. Judging by app_nunchuk.c, which already does this, I assume it is fine, but there may be some application for PPM_CTRL_TYPE_NONE that requires continuing through the loop that I am not aware of. If so, just let me know and I can generalize it to simply store the value of ppm_servo_val once it's calculated, and save the extra needed logic for custom firmware.